simplebgc/src/commands/read_params.rs

```diff
@@ -51,8 +51,8 @@ payload_rpy!(AxisRcParams, 8);
 #[repr(u8)]
 pub enum AxisRcMode {
     Angle = 1 << 0,
-    Speed = 1 << 1,
-    Inverted = 1 << 2,
+    Fixed = 1 << 1,
+    Inverted = 1 << 3,
 }
```

Comment:
were you able to determine whether the `Angle` mode corresponds to bits 0..2 = `000` or whether it corresponds to bit 0 = 1?

Reply:
So the value of the first three bits corresponds to whether it's `AngleRegular`, `AngleTracking` or `Speed`. The fourth bit is a flag for whether the mode is inverted or not.